src/Statistics.java

```diff
@@ -13,6 +14,7 @@
 	 */
 	public static double average(double[] x) {
```

Comment:
In this method, is it better to create a variable for x.length ?? because you use it many times.

https://github.com/NokKbl/UnitTestingExercise/blob/4a0de63946741f42929f61099b73c0fa073ad9a9/src/Statistics.java#L15-L20